### PR TITLE
Add Pgx Tab FAQ to App

### LINKED ISSUE #219: Add PGx Tab & FAQ to App

### PR DESCRIPTION
**Pull request by Benjamin-Frost**
Closes #219

### CODE REVIEW

File: app/lib/common/widgets/app.dart [hunk 1]
```diff
@@ -10,8 +9,8 @@ class PharmeApp extends StatelessWidget {
   PharmeApp({Key? key}) : super(key: key);
 
   final _appRouter = AppRouter();
-  final _isLoggedIn =
-      Hive.box('preferences').get('isLoggedIn', defaultValue: false) as bool;
+  final _isLoggedIn = true;
+  // Hive.box('preferences').get('isLoggedIn', defaultValue: false) as bool;
```
Comment (kolioOtSofia) [inline app/lib/common/widgets/app.dart:13]: Most prolly you wanna undo this change :D

File: app/lib/pgx/widgets/expansion_panel_item.dart [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  ExpansionPanelItem({
+    required this.headerValue,
+    required this.expandedValue,
+    this.isExpanded = false,
+  });
+
+  String headerValue;
+  String expandedValue;
+  bool isExpanded;
+}
```
Comment (kolioOtSofia) [inline app/lib/pgx/widgets/expansion_panel_item.dart:11]: Seems like you dont need this file

**Review by kolioOtSofia — CHANGES_REQUESTED** [review]

**Review by kolioOtSofia — APPROVED** [review]

**Review by dieknolle3333 — APPROVED** [review]
